packages/apps/client/src/lib/mdParser/index.ts

```diff
@@ -24,6 +27,13 @@ export class ParserStateImpl implements ParserState {
 		})
 		this.buffer = ''
 	}
+	setMarker = () => {
```

Comment:
Maybe this should be called `pushLeaf` (i.e. a node that has no children) and accept an argument of `NodeBase`? I was expecting `setMarker` to have something to do with ProseMirror markers.